seed/challenges/01-front-end-development-certification/object-oriented-and-functional-programming.json

```diff
@@ -563,10 +563,11 @@
       "id": "cf1111c1c16feddfaeb3bdef",
       "title": "Concatenate Arrays with .concat",
       "description": [
-        "<code>concat</code> can be used to merge the contents of two arrays into one.",
+        "<code>concat</code> can be used to merge the contents of two or more arrays into one.",
         "<code>concat</code> takes an array as an argument and returns a new array with the elements of this array concatenated onto the end.",
-        "Here is an example of <code>concat</code> being used to concatenate <code>otherArray</code> onto the end of <code>oldArray</code>:",
-        "<code>newArray = oldArray.concat(otherArray);</code>",
+        "Here are examples of <code>concat</code> being used to concatenate array(s):",
+        "Ex. 1:\n<code>otherArray</code> onto the end of <code>oldArray</code>:\n<code>newArray = oldArray.concat(otherArray);</code>",
+        "Ex. 2:\n<code>newArray</code> with all values of <code>arr1, arr2,</code> and <code>arr3</code>:\n<code>newArray = arr1.concat(arr2, arr3);</code>",
```

Comment:
@Bouncey I've made the requested changes, but was having issues with the branch name so I had to delete my branch name and create a new one. Can I resubmit another PR?